cmd/junod/cmd/forceprune.go

```diff
@@ -3,7 +3,7 @@ package cmd
 // DONTCOVER
```

Comment:
This breaks in SDK v46/47 yes? test it! - if so need to remove force prune entirely